### PR TITLE
tweaks to geocoding code

### PR DESCRIPTION
**Pull request by corybrunson**
Hi @emily-noble, these are some small edits that made the code easier for me to read and execute. Do the look OK and work for you? (I forgot to mention in the commit log that i introduced a breaking change: `inputPath` is not prespecified in the `executeInteractiveScript()` function, so the user is no longer prompted to enter `" "` to default to the sample file.)

I'm not sure if a second-order PR is the right way to go, but it's at least possible because the original repo isn't mine!

Medium-term, if you want to expand this to a self-contained package, i would be glad to help organize it! For now, it seems to do want we want. It's running on my device now, and i'll check in tomorrow after it's finished.

### CODE REVIEW

File: data-raw/geocode-data/geocode.r [hunk 1]
```diff
@@ -204,17 +217,22 @@ processFile <- function(inputFilePath, outputFilePath, email)
 #'
 #' Interactive mode will prompt the user for input files and email.
 executeInteractiveScript <- function() {
-  email = readline(
-    "Please provide your email. We need this to comply with terms of service for the nominatim geocoder). "
+  cat(
+    "Please provide your email.\n",
+    "We need this to comply with terms of service for the nominatim geocoder).",
```
Comment (emily-noble) [inline data-raw/geocode-data/geocode.r]: I noticed there's a close parenthesis here that's not needed.

File: data-raw/geocode-data/geocode.r [hunk 2]
```diff
@@ -204,17 +217,22 @@ processFile <- function(inputFilePath, outputFilePath, email)
...
   
-  downloadDryadData = readline(
-    "Do you want to download data from Dryad Data? Y = download the data directly from datadryad.org, N = enter your own file path. "
+  cat(
+    "Do you want to download data from Dryad Data?\n",
+    "Y = download the data directly from datadryad.org,\n",
+    "N = enter your own file path.",
+    sep = ""
   )
+  downloadDryadData <- readline("Downlead? ")
```
Comment (emily-noble) [inline data-raw/geocode-data/geocode.r]: Download is misspelled

**Review by emily-noble — CHANGES_REQUESTED** [review]
Looks like there's a few minor typos that could be fixed (some of them my fault!)

File: data-raw/geocode-data/geocode.r (hunk 2 above)
Comment (corybrunson) [reply data-raw/geocode-data/geocode.r]: Is it correct etiquette for me to "resolve" the conversation once i've pushed corrections?

**Pull request merged**